authorization/src/main/java/io/deephaven/auth/codegen/impl/TableServiceContextualAuthWiring.java

```diff
@@ -0,0 +1,433 @@
+//
+// Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+//
+// ---------------------------------------------------------------------
+// This class is generated by GenerateContextualAuthWiring. DO NOT EDIT!
+// ---------------------------------------------------------------------
+//
+package io.deephaven.auth.codegen.impl;
+
+import io.deephaven.auth.AuthContext;
+import io.deephaven.auth.ServiceAuthWiring;
+import io.deephaven.engine.table.Table;
+import io.deephaven.proto.backplane.grpc.ApplyPreviewColumnsRequest;
+import io.deephaven.proto.backplane.grpc.AsOfJoinTablesRequest;
+import io.deephaven.proto.backplane.grpc.ComboAggregateRequest;
+import io.deephaven.proto.backplane.grpc.CreateInputTableRequest;
+import io.deephaven.proto.backplane.grpc.CrossJoinTablesRequest;
+import io.deephaven.proto.backplane.grpc.DropColumnsRequest;
+import io.deephaven.proto.backplane.grpc.EmptyTableRequest;
+import io.deephaven.proto.backplane.grpc.ExactJoinTablesRequest;
+import io.deephaven.proto.backplane.grpc.ExportedTableUpdatesRequest;
+import io.deephaven.proto.backplane.grpc.FetchPandasTableRequest;
+import io.deephaven.proto.backplane.grpc.FetchTableRequest;
+import io.deephaven.proto.backplane.grpc.FilterTableRequest;
+import io.deephaven.proto.backplane.grpc.FlattenRequest;
+import io.deephaven.proto.backplane.grpc.HeadOrTailByRequest;
+import io.deephaven.proto.backplane.grpc.HeadOrTailRequest;
+import io.deephaven.proto.backplane.grpc.LeftJoinTablesRequest;
+import io.deephaven.proto.backplane.grpc.MergeTablesRequest;
+import io.deephaven.proto.backplane.grpc.NaturalJoinTablesRequest;
+import io.deephaven.proto.backplane.grpc.RunChartDownsampleRequest;
+import io.deephaven.proto.backplane.grpc.SelectDistinctRequest;
+import io.deephaven.proto.backplane.grpc.SelectOrUpdateRequest;
+import io.deephaven.proto.backplane.grpc.SnapshotTableRequest;
+import io.deephaven.proto.backplane.grpc.SortTableRequest;
+import io.deephaven.proto.backplane.grpc.Ticket;
+import io.deephaven.proto.backplane.grpc.TimeTableRequest;
+import io.deephaven.proto.backplane.grpc.UngroupRequest;
+import io.deephaven.proto.backplane.grpc.UnstructuredFilterTableRequest;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest;
+import io.deephaven.proto.backplane.grpc.WhereInRequest;
+import java.util.List;
+
+public interface TableServiceContextualAuthWiring extends ServiceAuthWiring {
+    void checkPermissionGetExportedTableCreationResponse(AuthContext authContext, Ticket request,
+            List<Table> sourceTables);
+
+    void checkPermissionFetchTable(AuthContext authContext, FetchTableRequest request,
+            List<Table> sourceTables);
+
+    void checkPermissionFetchPandasTable(AuthContext authContext, FetchPandasTableRequest request,
```

Comment:
Nothing calls this.

Reply:
We do not have this method implemented. It is also not part of Batch. @niloc132 should we remove this method from the API, or is it needed for future use?